### ISSUE TITLE
Can't move cells with mouse

### ISSUE THREAD

**Issue by Ssslakter** [microsoft/vscode-jupyter#15355]
Type: <b>Bug</b>

# Behaviour
## Expected vs. Actual

When I drag cell with mouse it moved to another position across other cells. Now it only works if I move my cell to the top of the file and only then will it be placed above the topmost cell.

## Steps to reproduce:

1. Drag some cell in jupyter notebook
2. move it to another place
3. drop it
4. the cell stays where it was before

# Diagnostic data

-   Python version (& distribution if applicable, e.g. Anaconda): 3.11.7
-   Type of virtual environment used (e.g. conda, venv, virtualenv, etc.): Conda
-   Value of the `python.languageServer` setting: Default

<details>

<summary>User Settings</summary>

<p>

```

languageServer: "Pylance"

```

</p>
</details>

Extension version: 2024.2.0
VS Code version: Code 1.87.2 (863d2581ecda6849923a2118d93a088b0745d9d6, 2024-03-08T15:20:17.278Z)
OS version: Windows_NT x64 10.0.19045
Modes:
Remote OS version: Linux x64 5.15.133.1-microsoft-standard-WSL2

<details>
<summary>A/B Experiments</summary>

```
vsliv368cf:30146710
vspor879:30202332
vspor708:30202333
vspor363:30204092
vscod805cf:30301675
binariesv615:30325510
vsaa593cf:30376535
py29gd2263:30899288
c4g48928:30535728
azure-dev_surveyone:30548225
962ge761:30959799
pythongtdpath:30769146
welcomedialogc:30910334
pythonidxpt:30866567
pythonnoceb:30805159
asynctok:30898717
pythontestfixt:30902429
pythonregdiag2:30936856
pyreplss1:30897532
pythonmypyd1:30879173
pythoncet0:30885854
pythontbext0:30879054
accentitlementst:30887150
dsvsc016:30899300
dsvsc017:30899301
dsvsc018:30899302
pydisww1:30981707
cppperfnew:30979542
d34g3935:30971562
fegfb526:30981948
bg6jg535:30979843
dsvsc020:30976470

```

</details>

**Comment by Ssslakter** [comment]
It just works now, after few updates

**Comment by halavins** [comment]
still doesn't work for me

**Comment by longtng** [comment]
using newest version of VSCode and still doesnt work for me.

**Comment by Ssslakter** [comment]
Yes, got the same effect again, it seems to be arbitrary stops and starts working

**Comment by david-waterworth** [comment]
I have the same issue, on Ubuntu 22.04 (scroll bars in general are annoying, way to small/hard to see etc - but I think that's Ubuntu's fault not vscode).

**Comment by DonJayamanne** [comment]
Please can you try the latest version of VS Code and let me know if this issue still persists.
Try version 1.88 of VS Code, thank you.
